tools/rector/config/rector.php

```diff
@@ -35,7 +35,11 @@
             '*/src/Entity/*',
         ],
         ChangeSwitchToMatchRector::class,
-        ReadOnlyPropertyRector::class,
+        ReadOnlyPropertyRector::class => [
+            'core-bundle/src/DependencyInjection/Filesystem/FilesystemConfiguration.php',
+            'core-bundle/src/Security/Exception/LockedException.php',
+            'core-bundle/src/Security/Authentication/Token/FrontendPreviewToken.php',
```

Comment:
I had to disable the rector for these files because it kept adding the `readonly` attribute even though the property cannot be readonly. 🙄